### PR TITLE
add benchmark (version 1.4.1)

### PR DESCRIPTION
**Pull request by Adsun701**
This is required to build the latest version of grpc.

### CODE REVIEW

File: mingw-w64-benchmark/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+package() {
+  cd "${srcdir}/build-${MINGW_CHOST}"
+  make DESTDIR="${pkgdir}" install
+  install -Dm644 "${srcdir}/${_realname}-${pkgver}/LICENSE" "${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}/LICENSE"
+  
+  local PREFIX_ROOT=$(cygpath -wm /usr)
+  # fix cmake config files
+  for _f in "${pkgdir}${MINGW_PREFIX}"/lib/cmake/${_realname}/*.cmake; do
+    sed -e "s|${PREFIX_ROOT}|/usr|g" -i ${_f}
```
Comment (Alexpux) [inline mingw-w64-benchmark/PKGBUILD]: Seem you copy-paste from Arch Linux. Mingw packages cant have /usr

**Comment by Alexpux** [comment]
@Adsun701 ping

**Comment by Adsun701** [comment]
Sorry for the delay; I have been doing a lot of work, but I will fix this issue as soon as I have time.

**Comment by Adsun701** [comment]
Ok. So I fixed issue. Now it should be fine.

**Pull request merged**